client/dashboard/me/tax-details/style.scss

```diff
@@ -0,0 +1,8 @@
+.user-tax-info__form {
+	flex-grow: 1;
+	max-width: 340px;
```

Comment:
I'm wondering why we want to set these `max-width`s here when we know the `<PageLayout>` is already set to "small", and so it seems like that would be enough to make sure these don't get too wide?

Reply:
I don't fully understand `PageLayout` and how to effectively use `VStack` and `HStack` and/or why the form is only taking up 25% of the space allowing the sidebar to take up 75%. For now, I made a quick fix in fd9e852 but want to revisit it and fix it properly.

Reply:
The problem was that the `VStack` that contained the component with a `DataForm` component needed `flex-shrink: 0`. Added this in ec86ef8.